src/RoleObjects/ButtonData.js

```diff
@@ -25,67 +27,68 @@ export default class ButtonData extends AccessibilityObject {
    * @inheritdoc
    */
   addChildAt(displayObject, index) {
-    if (!displayObject.accessible
+    if (
+      !displayObject.accessible
       || doesRoleUseSemanticallyInteractiveTag(displayObject.accessible.role)
-      || !_.isUndefined(displayObject.accessible.tabIndex)) {
+      || !_.isUndefined(displayObject.accessible.tabIndex)
+    ) {
       throw new Error(`Children of ${this.role} cannot be interactive`);
     }
     super.addChildAt(displayObject, index);
   }
 
   /**
-  * Sets whether the element should get pressed on page load
-  * @access public
-  * @param {boolean} val - true if button pressed
-  */
+   * Sets whether the element should get pressed on page load
```

Comment:
```suggestion
   * Sets whether the element should get pressed
```
some tech debt pointed out by @mnewcomb314 (`on page load` shouldn't be there)

Reply:
I think we can capture it as next task/tech debt, since this is not scoped for the current task

Reply:
honestly, I think this is the best time to nip these two comments in the bud - they are miscommunicating something that is untrue and removing these few words for clarity carries no regression risk and we're already touching this file anyway

also, IMO it's not worth the time, energy and effort to file and track a bug for something that can be fixed with two clicks of a mouse

Reply:
I agree @drosenberg123, you've also provided Suggestions that should be easy to apply.

Reply:
Changed as per the suggestion